packages/page-mixer/src/components/deposit/DepositConsole.tsx

```diff
@@ -14,7 +14,7 @@ import { CardRoot, CardSubTitle, CardTitle, DepositTitle, TriggerDeposit } from
 const depositLogger = LoggerService.get('Deposit');
 // todo get this by selecting assets
 export const DepositConsole: FC = () => {
-  const mixerInfos = useMixerInfos();
+  const mixerInfos = useMixerGroupsEntries();
```

Comment:
the naming here is now inconsistent, please also add types to variables so its clear to you as well as the reader.